### ISSUE TITLE
Different responseStrategy per regex

### ISSUE THREAD

**Issue by JordyMoos** [NekR/offline-plugin#275]
Is it possible to have different response strategies per regex?
Like i want my assets to be cache-first.
But for calls to the api needs to be up to date so for them i would like network-first.

**Comment by NekR** [comment]
No, not yet. You can implement API caching manually outside of the plugin and use `cache-first` in the plugin for assets.

**Comment by nealoke** [comment]
@NekR how 

> No, not yet. You can implement API caching manually outside of the plugin and use `cache-first` in the plugin for assets.

@NekR how do you achieve this?

**Comment by GGAlanSmithee** [comment]
@nealoke given how old this issue is, I think it would be better if you first read through the docs, and then opened a new, more specified, issue regarding your actual problem if you haven't found a solution by then.
